fitbenchmarking/fitting/misc.py

```diff
@@ -46,8 +46,10 @@ def create_result_entry(problem, status, chi_sq, runtime, minimizer,
     """
 
     if 'fitFunction' in ini_function_def:
-        ini_function_def = ini_function_def.replace('fitFunction', problem.equation)
-        fin_function_def = fin_function_def.replace('fitFunction', problem.equation)
+        ini_function_def = ini_function_def.replace(
```

Comment:
Feel free to disagree, but I think this is more readable with:
```
fin_function_def = fin_function_def.replace('fitFunction',
                                            problem.equation)
```

Reply:
Sorry I disagree 😞, plus what I have is closer to the [black](https://github.com/psf/black) formatting that is becoming more popular in Python

Reply:
No problem, I'll resolve this